src/formatDate.js

```diff
@@ -8,7 +8,32 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateArr = date.split(fromFormat[fromFormat.length - 1]);
+  const result = [];
```

Comment:
Use a more descriptive name for the variable, such as `formattedDate`, so that it is immediately clear what the variable is.